src/App.js

```diff
@@ -12,6 +13,13 @@ const error = debug('team-link:error');
 class App {
     constructor() {
         this.app = express();
+
+        this.app.use((req, res, next) => {
```

Comment:
Don't set headers (especially security ones) in a global middleware. If you really don't want to repeat yourself, make a function that will set the headers, and call it for each request where it is needed.